modules/nf-core/sylph/sketch/meta.yml

```diff
@@ -0,0 +1,60 @@
+name: "sylph_sketch"
+description: Sketching/indexing sequencing reads
+keywords:
+  - sketch
+  - metagenomics
+  - sylph
+  - indexing
+tools:
+  - sylph:
+      description: Sylph quickly enables querying of genomes against even low-coverage
+        shotgun metagenomes to find nearest neighbour ANI.
+      homepage: https://github.com/bluenote-1577/sylph
+      documentation: https://github.com/bluenote-1577/sylph
+      tool_dev_url: https://github.com/bluenote-1577/sylph
+      doi: 10.1038/s41587-024-02412-y
+      licence: ["MIT"]
+      identifier: biotools:sylph
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'test', single_end:false ]`
+    - reads:
+        type: file
+        description: Input fastq files
+        pattern: "*.{fq,fastq,fq.gz,fastq.gz}"
+  - - reference:
+        type: file
+        description: Reference genome file in FASTA format
+output:
+  - sketch_fastq_genome:
+      - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. [ id:'test', single_end:false ]
+          pattern: "my_sketches/*.sylsp"
+      - my_sketches/*.sylsp:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. [ id:'test', single_end:false ]
+          pattern: "my_sketches/*.sylsp"
+      - database.syldb:
+          type: map
```

Comment:
emiting only `sketch_fastq_genome`. if want to emit both, separate the outputs and emits

Reply:
I am a bit confused about the structure here. 

For example, in [kraken2](https://github.com/nf-core/modules/blob/master/modules/nf-core/kraken2/kraken2/main.nf#L20): 

`tuple val(meta), path('*report.txt')                           , emit: report`

and in [meta.yaml](https://github.com/nf-core/modules/blob/master/modules/nf-core/kraken2/kraken2/meta.yml#L77-L88): 

both `meta` and `report` are emitted.

Reply:
ya just thought it might be better to separate the outputs. that is ok, upto you

Reply:
Will leave it as it is for now.